Makefile

```diff
@@ -152,15 +142,11 @@ prod-push:
 
 # Create YAML file for deployment into CI
 ci-yaml: | $(KUSTOMIZE)
-	@echo "updating kustomize image patch file for manager resource"
-	sed -i'' -e 's@image: .*@image: '"$(CI_IMG):$(VERSION)"'@' ./config/default/vsphere_manager_image_patch.yaml
-	cmd/clusterctl/examples/vsphere/generate-yaml.sh
+	VSPHERE_MANAGER_IMG=${CI_IMG}:${VERSION} cmd/clusterctl/examples/vsphere/generate-yaml.sh
 
 ci-image: generate fmt vet manifests
 	docker build . -t "$(CI_IMG):$(VERSION)"
 	docker build . -f cmd/clusterctl/Dockerfile -t "$(CLUSTERCTL_CI_IMG):$(VERSION)"
-	@echo "updating kustomize image patch file for manager resource"
-	sed -i.tmp -e 's@image: .*@image: '"$(CI_IMG):$(VERSION)"'@' ./config/default/vsphere_manager_image_patch.yaml
```

Comment:
Wasn't clear to me why we override the image after the image is built, is it to make `generate-yaml.sh` easier to run afterwards?

Reply:
you are right, this is redundant

Reply:
^ @figo?